### PR TITLE
Follow mode features

### PR DESCRIPTION
**Pull request by PascalEschbach**
Added several follow mode features such as:

- Backtrack visualization
- Reach node radius
- Route rejoining
- Display length/count of nodes

Added & fixed mumble interop.

### CODE REVIEW

File: Assets/Augmented Tyria/Scripts/Utils/MumbleCamera.cs [hunk 1]
```diff
@@ -16,7 +16,9 @@ private void Awake()
 
     private void Update()
     {
-        this.Camera.fieldOfView = this.Mumble.Link.GetIdentity().Fov * Mathf.Rad2Deg;
+		var identity = this.Mumble.Link.GetIdentity();
+		if( identity != null )
```
Comment (ideka) [inline Assets/Augmented Tyria/Scripts/Utils/MumbleCamera.cs:20]: When is the identity ever null? When the game isn't running?

Comment (PascalEschbach) [reply Assets/Augmented Tyria/Scripts/Utils/MumbleCamera.cs:20]: Exactly.

**Comment by ideka** [comment]
I skimmed this, looks good. I'll take a better look when I have more time.

File: Assets/Augmented Tyria/Scripts/Utils/MumbleCamera.cs (hunk 1 above)
Comment (ideka) [reply Assets/Augmented Tyria/Scripts/Utils/MumbleCamera.cs:20]: And does this single check prevent all exceptions when the game isn't running?

Comment (PascalEschbach) [reply Assets/Augmented Tyria/Scripts/Utils/MumbleCamera.cs:20]: As far as I could check, it did.

**Pull request merged**